src/commands/cmd_zset.cc

```diff
@@ -513,6 +520,8 @@ class CommandZRangeGeneric : public Commander {
       max_idx = 2;
     }
 
+    if (count == 0) zero_count_ = true;
```

Comment:
I think it'd be better to use `spec.count == 0` to check if the count is zero instead of adding a new variable.

Reply:
https://github.com/apache/incubator-kvrocks/blob/e89af5289d5c3d12c47628394c72f0d67e652800/src/commands/cmd_zset.cc#L552-L566
Ok, I will add three statements in here.